python/shark_turbine/dynamo/importer.py

```diff
@@ -401,14 +401,37 @@ def _import_torch_op_overload(
         for i, value in enumerate(operation.results):
             self._v[(node, i)] = value
 
-    def _import_argument(self, arg: NodeArgument) -> Value:
+    def _import_argument(self, arg: NodeArgument, loc: Location = None) -> Value:
```

Comment:
Nit: In MLIR by convention, we typically always include the Location first in the argument list for cases like this and make it non-optional.

Reply:
Sounds good, I've pushed an edit